domain/src/main/java/org/oppia/android/domain/exploration/ExplorationDataController.kt

```diff
@@ -10,7 +10,8 @@ import org.oppia.android.util.data.DataProviders
 import org.oppia.android.util.system.OppiaClock
 import javax.inject.Inject
 
-private const val EXPLORATION_DATA_PROVIDER_ID = "ExplorationDataProvider"
+private const val EXPLORATION_DATA_CONTROLLER_PROVIDER_ID =
```

Comment:
This naming is incorrect.

If you see the issue description, in point 3 it mentions that you have to use following format.

`<function_name>_<optional_context>_PROVIDER_ID`

Which means it should be 

`GET_EXPLORATION_BY_ID_PROVIDER_ID`

Reply:
Fixed.